### PR TITLE
[METRICS SDK] Fix OTLP gRPC Metrics env variables

### PR DESCRIPTION
**Pull request by lalitb**
Fixes # (issue)

## Changes

Please provide a brief description of the changes here.

otlp gRPC Metrics exporter is reusing the env variable processing from otlp trace exporter, which is making it use env variables containing `_TRACE_` instead of `_METRICS_`. This is fixed now.

For significant contributions please make sure you have completed the following items:

* [ ] `CHANGELOG.md` updated for non-trivial changes
* [ ] Unit tests have been added
* [ ] Changes in public API reviewed

### CODE REVIEW

File: exporters/otlp/include/opentelemetry/exporters/otlp/otlp_environment.h [hunk 1]
```diff
@@ -313,6 +315,50 @@ inline OtlpHeaders GetOtlpDefaultMetricHeaders()
...
+    ssl_cert_path = opentelemetry::sdk::common::GetEnvironmentVariable(kOtlpSslCertificate);
+  }
+  return ssl_cert_path.size() ? ssl_cert_path : "";
+}
+
+inline const std::string GetOtlpDefaultMetricsSslCertificateString()
+{
+  constexpr char kOtlpTracesSslCertificateString[] =
+      "OTEL_EXPORTER_OTLP_METRICS_CERTIFICATE_STRING";
+  constexpr char kOtlpSslCertificateString[] = "OTEL_EXPORTER_OTLP_CERTIFICATE_STRING ";
```
Comment (ThomsonTan) [inline exporters/otlp/include/opentelemetry/exporters/otlp/otlp_environment.h:352]: Is there a trailing space in this string liternal?

Comment (lalitb) [reply exporters/otlp/include/opentelemetry/exporters/otlp/otlp_environment.h]: Good catch, it is copy-paste error from traces. Had fixed for traces but forgot to do it here. Have fixed it now. Thanks.

File: exporters/otlp/include/opentelemetry/exporters/otlp/otlp_environment.h [hunk 2]
```diff
@@ -313,6 +315,50 @@ inline OtlpHeaders GetOtlpDefaultMetricHeaders()
...
+  {
+    return true;
+  }
+  return false;
+}
+
+inline const std::string GetOtlpDefaultMetricsSslCertificatePath()
+{
+  constexpr char kOtlpMetricsSslCertificate[] = "OTEL_EXPORTER_OTLP_METRICS_CERTIFICATE";
+  constexpr char kOtlpSslCertificate[]        = "OTEL_EXPORTER_OTLP_CERTIFICATE ";
```
Comment (ThomsonTan) [inline exporters/otlp/include/opentelemetry/exporters/otlp/otlp_environment.h:338]: 1 more trailing space?

Comment (lalitb) [reply exporters/otlp/include/opentelemetry/exporters/otlp/otlp_environment.h]: oh bad. Thanks for noticing it, fixed now :(

**Review by ThomsonTan — APPROVED** [review]

File: exporters/otlp/include/opentelemetry/exporters/otlp/otlp_environment.h [hunk 3]
```diff
@@ -267,7 +267,9 @@ inline OtlpHeaders GetOtlpDefaultLogHeaders()
   return result;
 }
 
-inline const std::string GetOtlpDefaultHttpMetricEndpoint()
+// --- Metrics Environment Variables
+
```
Comment (esigo) [inline exporters/otlp/include/opentelemetry/exporters/otlp/otlp_environment.h:271]: nit: line break

**Review by esigo — APPROVED** [review]
LGTM
Thanks for the fix :)

File: exporters/otlp/include/opentelemetry/exporters/otlp/otlp_environment.h (hunk 3 above)
Comment (lalitb) [reply exporters/otlp/include/opentelemetry/exporters/otlp/otlp_environment.h]: fixed. thanks.

**Pull request merged**